TestApi/TestApi.DAL/Commands/AllocationService.cs

```diff
@@ -170,19 +175,32 @@ private async Task<Guid> CreateNewUserInTestApi(UserType userType, Application a
             );
 
             await _commandHandler.Handle(createNewUserCommand);
+        }
 
-            return createNewUserCommand.NewUserId;
+        private async Task<User> GetUserIdByUserTypeApplicationAndNumber(UserType userType, Application application, int number)
+        {
+            var getUserIdByUserTypeApplicationAndNumberQuery = new GetUserByUserTypeAppAndNumberQuery(userType, application, number);
```

Comment:
Bit picky here but looks a bit lazy to name a variable the same as the method name as long as it is rather than what it is.

Reply:
Renamed it to query

Reply:
Looks nicer too :)